src/components/CommentCreateForm/CommentCreateForm.tsx

```diff
@@ -60,13 +60,14 @@ const CommentCreateForm: React.FC<CommentCreateFormProps> = ({
 
     const onCommentChange = useCallback(
         ({ description }: { description: string }) => {
+            setDescription(description);
             onChange?.({ description, stateId: pushState?.id });
         },
         [onChange, pushState?.id],
     );
 
     const onCommentSubmit = useCallback(
-        async (form: CommentSchema) => {
+        (pushState?: { id: string }) => async (form: CommentSchema) => {
```

Comment:
Wierd implementation, why is function returns function? Make the union form data argument with optional `stateId` field